open_spiel/bots/stockfish/stockfish_bot.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 DeepMind Technologies Ltd. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#ifndef OPEN_SPIEL_BOTS_STOCKFISH_STOCKFISH_BOT_H_
+#define OPEN_SPIEL_BOTS_STOCKFISH_STOCKFISH_BOT_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "open_spiel/spiel_bots.h"
+#include "open_spiel/games/chess.h"
+#include "open_spiel/bots/stockfish/stockfish/src/types.h"
+#include "open_spiel/bots/stockfish/stockfish/src/thread.h"
+
+namespace PSQT {
+void init();
+}
+
+namespace open_spiel {
+namespace stockfish {
+
+class StockfishBot : public Bot {
+ public:
+  explicit StockfishBot(int argc, char **argv, int move_time);
+  virtual ~StockfishBot() { Threads.set(0); };
```

Comment:
Uhm. Not sure if this is good. Why it's here and are you sure it doesn't throw exceptions?